dataland-frontend/src/components/forms/parts/elements/derived/ActivityTree.ts

```diff
@@ -0,0 +1,970 @@
+import { type TreeNode } from "primevue/tree";
```

Comment:
General question: Where does this file get generated? (Or is it hand-written?)

Reply:
As far as i know it was manually created

Reply:
Fair enough. Nevertheless, there should be some documentation on where this data came from.

Reply:
There is no script for this? How do we keep it up to date in case there are changes? Manually?

Reply:
Until now manually. It was created when we were not sure how to proceed with the dev tools and alot of the other frontend DataModel files were also heavily modified by hand. But i agree this should be changed to an automatic process.

Reply:
I have added the files used to create this to the dev repo under the subproject dataland-activity-tree-builder.